components/Blog/blog.scss

```diff
@@ -0,0 +1,61 @@
+$side-nav-bg-color: #f9f9f9;
```

Comment:
Go ahead and put these in the `colors.scss` file and make them more descriptive of their color vs. their function in case we want to reuse them in the app.